### PR TITLE
Fix options migration for Dread's export params

### PR DESCRIPTION
**Pull request by henriquegemignani**

### CODE REVIEW

**Review by Miepee — APPROVED** [review]

File: randovania/interface_common/persisted_options.py [hunk 1]
```diff
@@ -196,7 +202,7 @@ def _only_new_fields(options: dict) -> dict:
     _only_new_fields,  # added DebugConnectorBuilder's layout_uuid
     _only_new_fields,  # added Dread's music sliders
     _only_new_fields,  # added Dread's "Access Permanently Closed" alt-shield
-    _only_new_fields,  # added Dread Ryujinx export to Unix systems
+    _dread_linux_ryujinx_path,  # added Dread Ryujinx export to Unix systems
```
Comment (Miepee) [inline randovania/interface_common/persisted_options.py:205]: actually, will this not screw people over who already migrated?

Comment (henriquegemignani) [reply randovania/interface_common/persisted_options.py:205]: Yes, but dev build ahoy

Comment (henriquegemignani) [reply randovania/interface_common/persisted_options.py:205]: Though only if they've migrated _and_ made some change that saves.

Comment (Miepee) [reply randovania/interface_common/persisted_options.py:205]: fair nuff

**Pull request merged**